Makefile

```diff
@@ -27,9 +27,15 @@ docker-compose-build:
 .PHONY: review
 review: test-cluster ## Specify review AKS environment
 	# PULL_REQUEST_NUMBER is set by the GitHub action
-	$(if $(PULL_REQUEST_NUMBER), , $(error Missing environment variable "PULL_REQUEST_NUMBER"))
+	$(if $(or ${PULL_REQUEST_NUMBER}, , ${PR_NUMBER}), , $(error Missing environment variable "PULL_REQUEST_NUMBER"))
```

Comment:
If we write the PULL_REQUEST_NUMBER to env, we shouldn't need to make changes in the Makefile. see https://github.com/DFE-Digital/register-early-career-teachers-public/pull/641